packages/ui/src/views/landing-page.tsx

```diff
@@ -15,66 +15,67 @@ interface TypesSpace {
   updated?: number
 }
 
-interface LandingPageProps {
+export interface LandingPageProps {
   spaces: TypesSpace[]
   useTranslationStore: () => TranslationStore
+  getProjectPath: (spaceId?: string) => string
+  createProjectLinkProps: LinkProps
 }
 
-export const LandingPageView: React.FC<LandingPageProps> = ({ spaces, useTranslationStore }) => {
-  const navigate = useNavigate()
+export const LandingPageView: React.FC<LandingPageProps> = ({
```

Comment:
Looks like `React` isn't imported in this file, so this will cause a lint warning. Can we directly import `FC` instead?
```suggestion
export const LandingPageView: FC<LandingPageProps> = ({
```